packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -277,8 +279,14 @@ class TransactionService {
            * Capture the TBT as span after observing for all long task entries
            * and once performance observer is disconnected
            */
-          if (tr.captureTimings && metrics.tbt.duration > 0) {
-            tr.spans.push(createTotalBlockingTimeSpan(metrics.tbt))
+          if (tr.captureTimings) {
+            if (metrics.tbt.duration > 0) {
+              tr.spans.push(createTotalBlockingTimeSpan(metrics.tbt))
+            }
+            /**
+             * TODO: define the proper place to store cls.
+             */
```

Comment:
We can report only non 0 values, it's wasteful to report when there is no layout shift.

Reply:
We're going to differentiate between supported and unsupported browsers but still send 0 if supported!